examples/MediaServices/vodworkflow_aes.php

```diff
@@ -331,7 +331,7 @@ function publishEncodedAsset($restProxy, $encodedAsset)
     $locator = $restProxy->createLocator($locator);
 
     // 6.4 Create a Smooth Streaming base URL
-    $stremingUrl = $locator->getPath().$manifestFile->getName().'/manifest';
+    $stremingUrl = $locator->getPath().$manifestFile->getName()."/manifest";
```

Comment:
It's not, but we went forced to applying many patches because someone has refactored the entire repository, this one seems that is stowaway from the older branch.